pom.xml

```diff
@@ -13,7 +13,7 @@
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
     <!-- 27-Mar-2025, tatu: Updated to later but not Latest available Quarkus -->
-    <quarkus.platform.version>3.15.4</quarkus.platform.version>
+    <quarkus.platform.version>3.20.0</quarkus.platform.version>
```

Comment:
Any particular reason for this version? Latest seems to be 3.24.4.
(perhaps some compatibility issue?)

Regardless, please update comment above wrt timing.